packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2429,6 +2429,56 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 ```
 
+### Clear Current User Sessions
```

Comment:
I like this title, so all good on my side
Can you double check with Damian as well please? Want to be sure it is clear that this is for current user, and that there is no confusion with the next topic right below this, which is for any user sessions (by admin or current user)

Reply:
@mauriciocastillosilva-okta yeah, I sync'ed with doc team, what do you think we move this section under `User Sessions` right below? So all session operations are doc'ed together?

Reply:
talked to Yuming offline, we will keep it here since it is under `me/lifecycle`